.env.development

```diff
@@ -0,0 +1,10 @@
+URL=https://app.outline.dev:3000
```

Comment:
Hmm, had to update this to run the app successfully.

Reply:
Can you be more specific – in what way did you have to update it? You're using a different URL?

Reply:
Yeah, mine's `https://local.outline.dev:3000`.

Reply:
I'll update the default here to that 👍

Reply:
How about leaving it empty and throwing instead? Letting the user know they need to update this...But, there may be many others I guess 🤔

Reply:
I want a standardized default that works for development here